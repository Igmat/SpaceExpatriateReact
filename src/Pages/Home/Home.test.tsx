import { Home } from ".";
import { RouterProvider as Router, createMemoryRouter } from "react-router-dom";
import renderer from 'react-test-renderer';

test('renders Home', () => {

  const router = createMemoryRouter([{element:<Home/>, path:"/"}])

  const tree = renderer.create(
      <Router router={router} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

