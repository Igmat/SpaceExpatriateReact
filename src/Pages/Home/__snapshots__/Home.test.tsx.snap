// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home 1`] = `
<div
  className="wrapper"
>
  <a
    className=""
    href="/game/dd20421b-4c5f-40ca-a1d9-301dd44d6fcb"
    onClick={[Function]}
  >
     
    <div
      className="start"
    >
      START GAME
    </div>
  </a>
</div>
`;
